Apps/Contoso.Android.Puppet/Properties/AndroidManifest.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I am curious what is the change here? Is it a change of UTF-8 to UTF-8 BOM encoding automatically added by Visual Studio? If so, it's best practice not to add them.

Reply:
@dhei From the desc of the PR:

> Fixes xml files by adding <U+FEFF> at the beginning of files.

All these changes are generated by VS for mac

Reply:
Aha, that's okay then. Sorry for failing to read the description.